docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.9'
+
+volumes:
+  fooocus-data:
+
+services:
+  app:
+    build: .
+    image: whitehara/fooocus
```

Comment:
I would suggest changing the image name to something more generic. Maybe plainly `fooocus`?